frontend/public/components/factory/details.tsx

```diff
@@ -49,53 +81,70 @@ export const DetailsPage = withFallback<DetailsPageProps>(({ pages = [], ...prop
         })),
     [resourcePageExtensions, props],
   );
-
+  const kindObj = props.kindObj ?? modelFor(props.kind);
+  const resolvedBreadcrumbExtension = useBreadCrumbsForDetailPage(kindObj);
+  const onBreadcrumbsResolved = React.useCallback((breadcrumbs) => {
+    setPluginBreadcrumbs(breadcrumbs ? () => breadcrumbs : undefined);
```

Comment:
Why are you setting `pluginBreadcrumbs` with `() => breadcrumbs` when you're sending it to `breadcrumbs` prop of `PageHeading`? The type of `breadcrumbs` prop is `{ name: string; path: string }[];`

Reply:
fixed it